### PR TITLE
fix(check-coverage): use 'ava' in 'cover'

### LINKED ISSUE #9: npm run check-coverage not working on Windows
This command always fails with `ERROR: no coverage files found`.
I found [this nyc issue](https://github.com/bcoe/nyc/issues/190) but there they only talk about Node v5.8.0 and NPM v3.7.3
My setup = Node v5.9.0 and NPM v3.8.1

### PR DESCRIPTION
**Pull request by sarbbottam**
Closes #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "precommit": "npm-run-all --parallel lint cover build",
```
Comment (kentcdodds) [inline package.json]: Could you also change this to:

```
"precommit": "npm-run-all --parallel lint cover build --serial check-coverage"
```

Thanks!

**Comment by kentcdodds** [comment]
@nyrosmith, once @sarbbottam responds to my comment (which I imagine will happen quickly based on my limited experience with @sarbbottam :smile:) could you validate this? It looks good to me so if it works for you, then feel free to merge it!

**Comment by sarbbottam** [comment]
Oh no. I again miss typed the issue # :cry:

**Comment by kentcdodds** [comment]
Cool, LGTM. @nyrosmith? Take your time :-)

File: package.json (hunk 1 above)
Comment (sarbbottam) [inline package.json]: @kentcdodds I bet you meant `--sequential` instead of `--serial` :smile:

Comment (kentcdodds) [inline package.json]: haha, yes, thank you!

**Comment by nyrosmith** [comment]
It works for me. Thx @sarbbottam

**Comment by kentcdodds** [comment]
Thanks @sarbbottam! I've given you commit access. Please continue to contribute via PRs and get them signed off by at least one other person. Thanks!

**Comment by sarbbottam** [comment]
Thanks for the access! Sounds good.

**Pull request merged**